docs/source/configuration/qos_configs/qos_client_config.rst

```diff
@@ -0,0 +1,53 @@
+.. _qos-client-configuration:
+
+QoS Client Configuration
+========================
+
+.. warning::
+
+    If you are using Cassandra, then you can use the AtlasDB QoS to rate-limit requests to Cassandra. Note that this
```

Comment:
This is a bit unclear -- what if I don't have Cassandra, can I still use it? Also I think the warning should only be about the QoS service being under development and testing and the rest should be normal text.